### PR TITLE
To do add remove

### PR DESCRIPTION
**Pull request by osbentech**
This is a to do list Application, it's main purpose is to add, store, edit added tasks and remove them at will.
Below are the function implemented ;

    - Implemented a function for adding a new task (add a new element to the array).
    - Implemented a function for deleting a task (remove an element from the array).
    - Implemented a function for editing task descriptions.
    - By default new tasks has the property completed set to false and the property index set to the value of the new array length 
    - Deleting a task updates all remaining items' indexes, so they represent the current list order and are unique
    - All changes to the To Do List are saved in local storage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,53 @@
-# To-Do-List
\ No newline at end of file
+![Microverse](https://img.shields.io/badge/Microverse-blueviolet) ![HTML](https://img.shields.io/badge/-HTML-orange) ![CSS](https://img.shields.io/badge/-CSS-blue) ![JavaScript](https://img.shields.io/badge/-JavaScript-yellow)
```
Comment (Ismailco) [inline README.md:1]: - [ ] The app is not working so I can't check the local storage or the functionality of the app, kindly fix it so the app is working properly
- [ ] [OPTIONAL] Kindly follow the app design as per the requirement guidelines 

Screenshot_20220820_061810

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+          empty.push(getting[i]);
+          localStorage.setItem('list', JSON.stringify(empty));
+        }
+        list.replaceChild(listText, editInput);
+        listText.textContent = editInput.value;
+        list.style.backgroundColor = '#fff';
+      }
+    });
+  });
+};
```
Comment (Ismailco) [inline src/index.js]: - [ ] kindly consider converting your code for editing tasks to a function so your app has a function to edit the tasks

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+    const result = getFromLocalStorage.filter((word) => word.description === listText.textContent);
+    const empty = [];
+    for (let i = 0; i < getFromLocalStorage.length; i += 1) {
+      if (result[0].description === getFromLocalStorage[i].description) {
+        continue;
+      }
+      empty.push(getFromLocalStorage[i]);
+    }
+    localStorage.setItem('list', JSON.stringify(empty));
+  });
```
Comment (Ismailco) [inline src/index.js]: - [ ] kindly consider converting your code for removing a task to a function so your app has a function to removing a task

File: src/index.js (hunk 2 above)
Comment (Ismailco) [inline src/index.js]: - [ ] Your app should have a new JavaScript file for the new functionality (add, remove, and edit), kindly consider creating a new file and moving your code there so you meet all the requirement

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @osbentech, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 3 above)
Comment (rotimiazeez) [reply src/index.js]: - Kindly see to this change it has not been implemented yet.

File: src/index.js (hunk 2 above)
Comment (rotimiazeez) [reply src/index.js]: - Kindly see to this change it has not been implemented yet

Comment (rotimiazeez) [reply src/index.js]: - Kindly see to this change it has not been implemented yet

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @osbentech , 
## Great work on making the changes 👏
 - Your app is working 💯
 - all features are implemented 😉
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._
- As suggested by the previous reviewer, kindly write a function for your features instead of pure DOM manipulation 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Benjamin,  

Your project is complete, great job! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Highlights 👍
- CRUD feature is functioning as intended

<br>
Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**